authorization/storage.go

```diff
@@ -43,7 +43,7 @@ func (s *Store) Update(ctx context.Context, fn func(kv.Tx) error) error {
 }
 
 func (s *Store) setup() error {
-	return s.Update(context.Background(), func(tx kv.Tx) error {
+	err := s.Update(context.Background(), func(tx kv.Tx) error {
```

Comment:
Did you make this change intending to add further changes in this code?

Reply:
I had changes there and backed them out. I will revert this.